src/applications/disability-benefits/all-claims/content/incidentDate.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+import { VaAdditionalInfo } from 'web-components/react-bindings';
```

Comment:
Can you explain why you are using `VaAdditionalInfo` on some of these and `va-additional-info` on others? Genuinely curious

Reply:
The `va-additional-info` web component does include generic analytics, but in the cases where I used `VaAdditionalInfo`, there were already custom analytics events, some conditional, in place. So I opted to disable the built-in analytics and continue using the custom events.